dss-collection-nft/contracts/DSSCollection.cdc

```diff
@@ -100,6 +104,8 @@ pub contract DSSCollection: NonFungibleToken {
                 self.id = slot.id
                 self.collectionGroupID = slot.collectionGroupID
                 self.logicalOperator = slot.logicalOperator
+                self.comparator = slot.comparator
```

Comment:
Should this be on the item instead?

Reply:
It works both places and there are trade-offs. On item is more flexible and you can do more with it. On slot it aligns with the other logic & keeps things simpler.

Went with the simpler approach as you could replicate some of the flexibility you gain by just having more slots.

Reply:
Updated to item.